lizard_auth_client/client.py

```diff
@@ -713,7 +713,7 @@ def get_billable_organisation(user, role_code=models.Role.BILLING_ROLE_CODE):
                 synced = True
 
 
-def sso_server_url(name):
+def sso_server_url(name, ignore_cache=False):
```

Comment:
Dit soort dingen kan je beter positief zeggen, dus `use_cache` i.p.v. `ignore_cache`. Want, wat doet de volgende code?

    if not ignore_cache == False:

Al snel krijg je in ieder geval "double negatives" als het al geen triple negatives zijn.

Zeker als de default False is, dan vertrouw ik mezelf niet meer dat ik de code correct lees :-)